src/main/java/com/redhat/cloud/policies/app/health/ProcSelfStatusExporter.java

```diff
@@ -41,27 +42,45 @@
 @ApplicationScoped
 public class ProcSelfStatusExporter {
 
+    private final Logger log = Logger.getLogger(this.getClass().getSimpleName());
+
     private static final String PATHNAME = "/proc/self/status";
 
     private boolean hasWarned = false;
 
-    long vmHwm;
-    long vmRss;
-    long rssAnon;
-    long rssFile;
-    long vmStk;
-    long vmLib;
-    long vmData;
-    long vmSize;
-    int threads;
+    @Inject
+    MeterRegistry registry;
+
+    AtomicLong vmHwm;
```

Comment:
`AtomicLong` isn't needed as there is no concurrency involved here. See [ProcSelfStatusExporter](https://github.com/RedHatInsights/policies-engine/blob/master/src/main/java/com/redhat/cloud/policies/engine/metrics/ProcSelfStatusExporter.java) from `policies-engine` (you could simply copy/paste that file here).